File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2949.py
a=0
elf=0
h=0
m=0
x=0

n = int(input())
for i in range(0, n):
    nome = input().split()
    e = nome[-1]
    if(e=='A'):
        a=a+1
    elif(e=='E'):
        elf=elf+1
    elif(e=='H'):
        h=h+1
    elif(e=='M'):
        m=m+1
    else :
        x=x+1

print("{} Hobbit(s)".format(x))
print("{} Humano(s)".format(h))
print("{} Elfo(s)".format(elf))
print("{} Anao(s)".format(a))
print("{} Mago(s)".format(m))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2949.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

a=0
elf=0
h=0
m=0
x=0

n = parseInt(lines.shift())
for(i=0;i<n;i++){
    nome = lines.shift()
    e = nome.split(' ')[1]
    if(e=='A') a++
    else if(e=='E') elf++
    else if(e=='H') h++
    else if(e=='M') m++
    else x++
}
console.log(x+" Hobbit(s)")
console.log(h+" Humano(s)")
console.log(elf+" Elfo(s)")
console.log(a+" Anao(s)")
console.log(m+" Mago(s)")
